### PR TITLE
Enable MPI tests for Python wrappers

### PR DESCRIPTION
**Pull request by dalg24**
sanity check doesn't do much but it does not hurt to have it...

### CODE REVIEW

File: python/test/CMakeLists.txt [hunk 1]
```diff
@@ -11,36 +11,43 @@ SET(PYTHON_TESTS
...
+        set(NUMBER_OF_PROCESSES_TO_EXECUTE ${ARGN})
     else()
-        ADD_TEST(
-            NAME ${TEST_NAME}_py
-            COMMAND ${PYTHON_EXECUTABLE} ${TEST_NAME}.py
-        )
-    endif()
-ENDFUNCTION()
+        set(NUMBER_OF_PROCESSES_TO_EXECUTE 1)
+    endiF()
```
Comment (Rombur) [inline python/test/CMakeLists.txt]: endiF -> endif

**Comment by Rombur** [comment]
Fix the typo and merge yourself

**Comment by dalg24** [comment]
@Rombur: Would you review the changes to CMake plz?

File: cmake/UnitTesting.cmake [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    target_link_libraries(${TEST_NAME}.exe ${Boost_CHRONO_LIBRARY})
+    set_target_properties(${TEST_NAME}.exe PROPERTIES
+        CXX_STANDARD 14
+        CXX_STANDARD_REQUIRED ON
+    )
+    if(ARGN)
+        set(NUMBER_OF_PROCESSES_TO_EXECUTE ${ARGN})
+    else()
+        set(NUMBER_OF_PROCESSES_TO_EXECUTE 1)
+    endiF()
```
Comment (Rombur) [inline cmake/UnitTesting.cmake]: endif -> endiF

**Comment by Rombur** [comment]
Fix the typo and merge

**Pull request merged**